From 9b53b5596a9165230ba8a231d0779d22e9c6b380 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 17 Sep 2014 16:51:43 -0400
Subject: [redhat] drm: hack to make drm patches apply

Since its apparently way too hard for one subsystem in particular to
submit proper patch series', I'm committing this as patch 0.5/7 for
the subsequent drm series, so that it'll apply without having to get
another multi-megabyte patch sent to the list or refreshed in patchwork.

This bit went in as part of a proper patchset from Myron Stowe, in patch
89302, "[acpi] core: Store an ACPI device pointer in struct acpi_dev_node",
aka RHEL7 commit 5e4bfe0a822167bd9a7f62ed8026ced9b8afadc3.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/radeon/radeon_atpx_handler.c b/drivers/gpu/drm/radeon/radeon_atpx_handler.c
index 27a6e9d..d96070b 100644
--- a/drivers/gpu/drm/radeon/radeon_atpx_handler.c
+++ b/drivers/gpu/drm/radeon/radeon_atpx_handler.c
@@ -8,7 +8,8 @@
  */
 #include <linux/vga_switcheroo.h>
 #include <linux/slab.h>
-#include <linux/acpi.h>
+#include <acpi/acpi.h>
+#include <acpi/acpi_bus.h>
 #include <linux/pci.h>
 
 #include "radeon_acpi.h"
-- 
1.7.1